spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -186,7 +185,7 @@ public void testCustomPartitionCountOverridesDefaultIfLarger() throws Exception
 		// Let the consumer actually bind to the producer before sending a msg
 		binderBindUnbindLatency();
 		moduleOutputChannel.send(message);
-		Message<?> inbound = moduleInputChannel.receive(2000);
+		Message<?> inbound = moduleInputChannel.receive((int)(2000 * timeoutMultiplier));
```

Comment:
Yeah, I know - but I was hoping we could pick up a value. Just looking through all of them, there doesn't seem to be any criterion by which those values are set - in most cases it's 1s and they're just bumped up for certain tests which I suppose were the ones that happened to fail at some point - e.g. Kafka tends to have larger values. But we already introduced the concept of per-binder test multiplier, so we can just start with different default values - wdyt?